### DISCUSSION TITLE
%E5%A6%82%E4%BD%95%E4%BD%BF%E7%94%A8-floating-ui-%E6%89%93%E9%80%A0-tooltip-%E7%BB%84%E4%BB%B6

### DISCUSSION THREAD

**Discussion by giscus** [n1203/blog#8] [Show and tell]
# %E5%A6%82%E4%BD%95%E4%BD%BF%E7%94%A8-floating-ui-%E6%89%93%E9%80%A0-tooltip-%E7%BB%84%E4%BB%B6

Floating-UI 是一个库，可帮助您创建“浮动”元素，例如工具提示、弹出框、下拉列表等。

https://laohu.notion.pet/%E5%A6%82%E4%BD%95%E4%BD%BF%E7%94%A8-floating-ui-%E6%89%93%E9%80%A0-tooltip-%E7%BB%84%E4%BB%B6

**Comment by Xuyan-cmd** [comment]
不错，下次来试试哈哈
